doc/input-output-reference/src/overview/group-zone-equipment.tex

```diff
@@ -68,6 +70,8 @@ \subsection{ZoneHVAC:AirDistributionUnit}\label{zonehvacairdistributionunit}
   \hyperref[airterminalsingleductparallelpiureheat]{AirTerminal:SingleDuct:ParallelPIU:Reheat}
 \item
   \hyperref[airterminalsingleductconstantvolumefourpipeinduction]{AirTerminal:SingleDuct:ConstantVolume:FourPipeInduction}
+\item
+  \hyperref[airterminalsingleductconstantvolumefourpipebeam]{AirTerminal:SingleDuct:ConstantVolume:FourPipeBeam}
```

Comment:
Added missing `AirTerminal:SingleDuct:ConstantVolume:FourPipeBeam` air terminal options of ADU.